includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -229,9 +228,7 @@ public function prepare_items() {
 		// Handle orderby.
 		$orderby = '';
 		if ( ! empty( $_GET['orderby'] ) ) {
-			if ( array_key_exists( esc_html( $_GET['orderby'] ), $this->get_sortable_columns() ) ) {
```

Comment:
This line was actually causing sorting not to work for emails and so. What was happening **array_key_exists** was fetching internal names for columns and **$_GET['orderby']** was returning the value that was sent in a query to order by DB column value.